modules/pel/peltool/README.md

```diff
@@ -0,0 +1,83 @@
+# peltool Usage
+
+To see all of peltool's capabilities, use the help option `peltool.py --help`.
+
+**Note:** For non-BMC environment user must pass the PEL directory to parse PELs
+          e.g. `peltool.py -p <pel_dir> -l`, `peltool.py -p <pel_dir> -n`,
+               `peltool.py -p <pel_dir> -a`
+
+#### List PELs
+
+- Get list of PELs: `peltool.py -l` (By default serviceable PELs are listed)
+- Get list of PELs including serviceable PELs: `peltool.py -ls`
+- Get list of PELs including non-serviceable PELs: `peltool.py -lN`
+- Get list of PELs including hidden PELs: `peltool.py -lH`
+- Get list of PELs including terminating PELs: `peltool.py -lt`
+- Get list of serviceable, non-serviceable, and hidden PELs: `peltool.py -lsNH`
+
+##### List PELs based on severity and only options
+
+- Get list of PELs based on severity value: `peltool.py -l -S <Severity>`
```

Comment:
This wording isn't quite right, because it also gets the PELs with the default severities.  You could almost just leave out this example because on the next one you explain that it includes different severities.